src/jmespath.net/Expressions/JmesPathFunctionExpression.cs

```diff
@@ -13,12 +13,6 @@ public class JmesPathFunctionExpression : JmesPathExpression
         private readonly JmesPathExpression[] expressions_;
         private readonly JmesPathFunction function_;
 
-        public JmesPathFunctionExpression(string name, params JmesPathExpression[] expressions)
-            : this(JmesPathFunctionFactory.Default, name, expressions)

```

Comment:
It is no longer to instantiate the `JmesPathFunctionFactory` as a static object because it needs to be passed some additional context at runtime.

Maybe we need to implement dependency injection to make this easier.

Reply:
In an other pull request.